src/Helpers/Time.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe } from 'riteway'
+import { minutesToMiliseconds, secondsToMiliseconds } from './Time'
+
+describe('Time', async (should: any) => {
+  describe('minutesToMiliseconds', async (should: any) => {
```

Comment:
Doing a nested describe format doesn't work because it's not truly nested. Check out the console output on success. `Time` is about in the middle of the screen with no tests. `minutesToMiliseconds` and `secondsToMiliseconds` are at the bottom.
![screenshot from 2018-08-20 18-00-56](https://user-images.githubusercontent.com/5175120/44374467-2b6d2d80-a4a3-11e8-84d3-83d53d9b25ff.png)

Reply:
thanks @kennylavender !!! I will fix